warehouse/templates/manage/manage_base.html

```diff
@@ -87,23 +102,25 @@ <h3 class="sidebar-section__title">Your account</h3>
           <span class="sr-only">close</span>
         </a>
         <div class="modal__body">
-          <h3 class="modal__title">{{ title }} {{ confirm_string }}?</h3>
-        <div class="callout-block callout-block--danger callout-block--bottom-margin no-top-margin">
-          <p>
-            <i class="fa fa-exclamation-triangle" aria-hidden="true"><span class="sr-only">Warning</span></i>
-            This action cannot be undone!
-            {% if confirm_name == "Version" %}
-            You will not be able to re-upload a new distribution of the same type with the same version number.
-            Consider making a new release or a <a href="https://www.python.org/dev/peps/pep-0440/#post-releases">post release</a> instead.
-            {% endif %}
-          </p>
-        </div>
-        {% set context = 'your' if confirm_name.lower() == 'username' else 'the' %}
-        {{ extra_description if extra_description else '' }}
-        <p>Confirm {{ context }} {{ confirm_name|lower }} to continue.</p>
-        {% set name = "confirm_" + confirm_name.lower().replace(' ', '_') %}
-        <label for="{{ name }}">{{ confirm_name }}</label>
-        <input name="{{ name }}" data-action="input->confirm#check" data-target="confirm.input" type="text" autocomplete="off" autocorrect="off" autocapitalize="off">
+          <h3 class="modal__title">{{ title }} {% if confirm_string_in_title %}{{ confirm_string }}{% endif %}?</h3>
```

Comment:
```suggestion
          <h3 class="modal__title">{{ title }}{% if confirm_string_in_title %} {{ confirm_string }}{% endif %}?</h3>
```
This is adding an extra space before the question mark when `confirm_string_in_title` is false.